### PR TITLE
Write to single zarr store

### PR DESCRIPTION
**Pull request by talonchandler**
Fixes https://github.com/mehta-lab/recOrder/issues/338.

Tested on all combinations of [Retardance + Orientation, Phase, Retardance + Orientation + Phase] and [2D, 3D]. All of the resulting zarrs can be drag-and-drop opened in napari with the recorder reader.

### CODE REVIEW

File: recOrder/acq/acquisition_workers.py [hunk 1]
```diff
@@ -942,39 +938,30 @@ def _save_imgs(self, birefringence, phase, meta=None):
...
+        if birefringence.ndim == 3:
+            birefringence = birefringence[:, np.newaxis, ...]  # CYX -> CZYX
+        birefringence = birefringence[np.newaxis, ...]  # CZYX -> TCZYX
+        dataset["0"] = birefringence
 
         if phase is not None:
-            name = (
-                f"PhaseSnap.zarr" if not prefix else f"{prefix}_PhaseSnap.zarr"
-            )
+            dataset.append_channel("Phase", resize_arrays=True)
```
Comment (ziw-liu) [inline recOrder/acq/acquisition_workers.py:959]: We used to have `Phase2D` or `Phase3D` channel names depending on the reconstruction algorithm. Do we want to change that behavior?

File: recOrder/acq/acquisition_workers.py [hunk 2]
```diff
@@ -942,39 +938,30 @@ def _save_imgs(self, birefringence, phase, meta=None):
...
+            if phase.ndim == 2:
+                phase = phase[np.newaxis, ...]  # YX -> ZYX
```
Comment (ziw-liu) [inline recOrder/acq/acquisition_workers.py:961]: Ellipsis is not need here.

Comment (ziw-liu) [reply recOrder/acq/acquisition_workers.py:961]: Also for the lines above.

File: recOrder/acq/acquisition_workers.py [hunk 3]
```diff
@@ -942,39 +938,32 @@ def _save_imgs(self, birefringence, phase, meta=None):
...
-                os.path.join(self.snap_dir, name),
-                layout="fov",
-                mode="w-",
-                channel_names=["Retardance", "Orientation", "BF", "Pol"],
-            ) as dataset:
-                if birefringence.ndim == 3:
-                    birefringence = birefringence[:, np.newaxis, ...]
-                dataset["0"] = birefringence[np.newaxis, ...]
-                dataset.zattrs["recOrder"] = meta
+        dataset = open_ome_zarr(
```
Comment (ziw-liu) [inline recOrder/acq/acquisition_workers.py:947]: Is this store closed somewhere else?

File: recOrder/acq/acquisition_workers.py (hunk 1 above)
Comment (talonchandler) [reply recOrder/acq/acquisition_workers.py]: Nice catch! Thank you.

File: recOrder/acq/acquisition_workers.py (hunk 2 above)
Comment (talonchandler) [reply recOrder/acq/acquisition_workers.py]: Thanks! I'm an unwitting ellipsis machine!

File: recOrder/acq/acquisition_workers.py (hunk 3 above)
Comment (talonchandler) [reply recOrder/acq/acquisition_workers.py]: Now closed!

**Comment by talonchandler** [comment]
Just tested again on hummingbird. Thanks @ziw-liu!

**Review by ziw-liu — APPROVED** [review]

**Pull request merged**